woocommerce/woocommerce-ios
# Dark Mode: update colors for Products tab

Products (minus stars UI) for #1555 

## Changes

- Added a `border` semantic color
- Added a `UILabel` extension helper to style an actionable text label
- Applied actionable text label style to `ProductDetailsViewModel`'s actionable cells
- Removed unused code `NoteTableViewCell` and its xib
- Updated colors in `ProductImageCollectionViewCell` and `ProductsTabProductTableViewCell`
- Updated colors for Product cell status
- Removed setting `contentView.backgroundColor` in `UITableViewCell`'s applyDefaultBackgroundStyle()` so that the highlighted state has a different background color

## Testing

- Launch the app
- Go to the Products tab --> the Product cell should look as expected, with the correct text color for products with "draft" and "pending reviews" status. the image border color and placeholder colors should also look as expected
- Tap a Product with an image --> the Product images on the top should have a visible border with the correct color

## Example screenshots

Screen | Dark | Light
-- | -- | --
Products tab | ![Simulator Screen Shot - iPhone 8 - 2019-12-09 at 18 17 55](https://user-images.githubusercontent.com/1945542/70427875-794ee000-1ab0-11ea-9936-45674f9027d1.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-09 at 18 18 03](https://user-images.githubusercontent.com/1945542/70427877-79e77680-1ab0-11ea-8e65-dee008d6f2d9.png)
Product Details | ![Simulator Screen Shot - iPhone 8 - 2019-12-11 at 10 25 54](https://user-images.githubusercontent.com/1945542/70585918-af9b7500-1c00-11ea-9514-99ce7b4d891f.png) | ![Simulator Screen Shot - iPhone 8 - 2019-12-11 at 10 26 02](https://user-images.githubusercontent.com/1945542/70585919-af9b7500-1c00-11ea-8cbb-47b97a2f3ac7.png)









Update release notes:
